test/dotcom/system_status/alerts_test.exs

```diff
@@ -0,0 +1,178 @@
+defmodule Dotcom.SystemStatus.AlertsTest do
+  use ExUnit.Case, async: true
+  doctest Dotcom.SystemStatus.Alerts
+
+  import Test.Support.Factories.Alerts.Alert
+
+  alias Dotcom.SystemStatus.Alerts
+
+  defp local_datetime(naive) do
```

Comment:
We usually put private functions at the bottom of the file, though, tbh, not sure we need this. See below.

Reply:
[Done!](https://github.com/mbta/dotcom/pull/2327/commits/73be207c966c6271993448a57321d1f0557c4cde)